src/app/pages/news/11.3/02-inscription-view.md

```diff
@@ -1,6 +1,20 @@
 ## Inscription View {#inscriptionView}
 
-Massive design update, browser, finalization ...
+The new Inscription View is finally feature complete! You are able to configure 
+all elements on the totally redesigned interface.
+
+- **UI update**:  Prepare to be amazed by the refreshed user interface! 
+Thanks to the sleek new design, navigating through inscriptions has 
+never been smoother, making oversight effortless. The UI update not only 
+attends to the needs of advanced users but also makes the functionality 
+more appealing for new users. Among other things, the redesign features 
+collapsible sections for every subpart, improved tab naming and overall interface streamlining.
+- **Browsers**: We added assisting browsers to search for CMS entries, DataClass attributes, 
+Java-Types and the like. These browsers make element configurations an easy and swift 
+user-experience without the need to write any code.
+- **Code Completion**: A notable improvement in this version is the completion 
+list within script/code fields. You now get more functions, types and attributes 
```

Comment:
types and attributes auto inserted -> types, and attributes auto-inserted (comma, dash)